Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 gem 'hydra', '~> 7.1.0'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 4.1.4'
+gem 'blacklight', github: 'projectblacklight/blacklight', branch: 'release-5.5'
```

Comment:
We were using 5.5.4. In the branch I specify, that the last node of that release branch is 5.5.4. You should be able to verify this with the command `gem list | grep blacklight`

FYI, this gives me the ability to use a local copy of Blacklight if it exists in the vendor/gems directory. If it does not exist, bundler will use what's in RubyGems.org.